packages/w3up-client/test/capability/blob.test.js

```diff
@@ -32,17 +35,30 @@ export const BlobClient = Test.withContext({
 
     const bytes = await randomBytes(128)
     const bytesHash = await sha256.digest(bytes)
-    const multihash = await alice.capability.blob.add(new Blob([bytes]))
+    const link = createLink(car.code, bytesHash)
+    const commitment = await alice.capability.blob.add(new Blob([bytes]), {
+      fetch: setupGetReceipt(function* () {
+        yield link
+      }),
+    })
 
     // TODO we should check blobsStorage as well
     assert.deepEqual(
-      await allocationsStorage.exists(space.did(), multihash.bytes),
+      await allocationsStorage.exists(
+        space.did(),
+        // @ts-ignore Element
```

Comment:
same as above, would be good to know more about this ignore, and use the expect-error to help out when it is not relevant anymore